### DISCUSSION TITLE
How to make a a split window into a full window without closing others

### DISCUSSION THREAD

**Discussion by NeotenicPrimate** [helix-editor/helix#7166] [Q&A] [answered]
I have my two windows open, I'd like to make one of them temporarily take up the full terminal without closing the other. Is this possible? If not, would this be a desirable feature?

I am aware of `o` and `C-o` in window mode (`C-w`), but this closes all other windows.

I've looked for quite a while but I am unable to find a question relating to this. Sorry, if I missed it.

**Comment by clo4** [comment, marked as answer]
I don't think this is possible right now. This sounds like what Vim calls tabs - there isn't really an equivalent in Helix right now. Probably the closest you'd get is using a terminal multiplexer eg. tmux or zellij and switching active windows/panes (someone please correct me if I'm wrong!)

**Reply by NeotenicPrimate** [reply]
Thanks! Yeah, I've been using Zellij and I love it. But if Helix had its equivalent of panes I wouldn't need anything but Helix, which would be amazing.

**Reply by kirawi** [reply]
https://github.com/helix-editor/helix/pull/7109
